app/routes/items/detail.js

```diff
@@ -40,6 +40,9 @@ export default AuthorizeRoute.extend({
     }
     await this.store.findAll("restriction", { reload: true });
     await this.store.findAll("donor_condition", { reload: true });
+    await this.store.findAll("processing_destinations_lookup", {
+      reload: true
+    });
```

Comment:
All 3 of those here probably don't need to be reloaded every time we open item details.
Either:
- We preload them
- We load them when we open the items details page, but only the first time (cache them essentially)

Reply:
cached it